lib/iris/tests/unit/coords/test_Coord.py

```diff
@@ -169,12 +169,12 @@ def test_all_inside(self):
     def test_points_inside_bounds_outside(self):
         lat = DimCoord([-80, 0, 70], units='degree', standard_name='latitude')
         lat.guess_bounds()
-        self.assertArrayEqual(lat.bounds, [[-120, -40], [-40, 35], [35, 105]])
+        self.assertArrayEqual(lat.bounds, [[-90, -40], [-40, 35], [35, 90]])
 
     def test_points_to_edges_bounds_outside(self):
         lat = DimCoord([-90, 0, 90], units='degree', standard_name='latitude')
         lat.guess_bounds()
-        self.assertArrayEqual(lat.bounds, [[-135, -45], [-45, 45], [45, 135]])
+        self.assertArrayEqual(lat.bounds, [[-90, -45], [-45, 45], [45, 90]])
```

Comment:
This test and the one above are no identical to their namesakes in `Test_guess_bounds__enabled_latitude_clipping` and can be removed.

Reply:
Although, perhaps we can to keep the tests but toggle the flag the other way?

Reply:
Question is, do we want to test deprecated behaviour?

Reply:
If we did test the deprecated behaviour we'd need to capture the deprecation warning that would also be raised. Speaking from the convention we've set up when updating the other FUTURE flags, we've tended to update tests to test the new behaviour if the test is still necessary, and remove the test entirely if not.

Reply:
I don't know, but one argument in favour of testing it is this: the reason it's deprecated rather than simply removed is so that users can still use it to give them a chance to change their code. If we don't test it and it breaks, we take away that grace period.